frontend/src/lib/components/MilestoneOverview.svelte

```diff
@@ -1,40 +1,39 @@
+<svelte:options runes={true} />
 <script lang="ts">
+import {
+	type MilestonePublic,
+	getCurrentMilestoneAnswerSession,
+} from "$lib/client";
+import AlertMessage from "$lib/components/AlertMessage.svelte";
 import CardDisplay from "$lib/components/DataDisplay/CardDisplay.svelte";
 import GalleryDisplay from "$lib/components/DataDisplay/GalleryDisplay.svelte";
-import { convertData, data } from "$lib/components/MilestoneOverview";
 import Breadcrumbs from "$lib/components/Navigation/Breadcrumbs.svelte";
+import { currentChild } from "$lib/stores/childrenStore.svelte";
 import { activeTabChildren } from "$lib/stores/componentStore";
+import { contentStore } from "$lib/stores/contentStore.svelte";
+import {
+	CheckCircleSolid,
+	ExclamationCircleSolid,
+	GridOutline,
+	RectangleListOutline,
+	UserSettingsOutline,
+} from "flowbite-svelte-icons";
+import { _, locale } from "svelte-i18n";
 
-export let milestones = data.milestones;
-
-interface MilestoneData {
-	header: string;
-	href: string;
-	summary: string;
-	auxilliary: string;
-	complete: boolean;
-	answer: string;
-}
-
-function searchStatus(data: MilestoneData[], key: string): MilestoneData[] {
+function searchStatus(data: any[], key: string): any[] {
 	if (key === "") {
 		return data;
 	} else {
 		return data.filter((item) => {
 			// button label contains info about completion status => use for search
-			if (key === "fertig") {
-				return item.complete === true;
-			} else if (key === "unfertig") {
-				return item.complete === false;
+			if (key === $_("milestone.complete")) {
+			} else if (key === $_("milestone.incomplete")) {
```

Comment:
is there some logic missing here?

Reply:
thanks. I indeed overlooked that at some point :grimacing: